tests/read_only_trx_test.py

```diff
@@ -16,9 +20,14 @@
 Print=Utils.Print
 errorExit=Utils.errorExit
 
+appArgs=AppArgs()
+appArgs.add(flag="--read-only-threads", type=int, help="number of read-only threads", default=0)
```

Comment:
Running test with default args fails with: `"--specific-num" provided value= 1 is higher than "--nodes" provided value=1`

Reply:
Sorry. Fixed it.